h2o-r/h2o-package/R/models.R

```diff
@@ -5054,3 +5054,20 @@ h2o.reset_threshold <- function(object, threshold) {
     return(NULL)
   }
 }
+
+#'
+#' Retrieve rule importances for a Rulefit model
+#'
+#' @param object An \linkS4class{H2OTable} object.
+#' @export
+h2o.get_rule_importance <- function(object) {
```

Comment:
Is the naming in line with the "universe" conventions we should follow ? snake case is fine I guess, but what about the `get` prefix ? @sebhrusen is the API guy.

Reply:
yesyes I put this into api changes channel and waiting for response there. Did this in accordance with ntrees_actual / h2o.get_ntrees_actual but may change according to response.

Reply:
yes, to make code more portable between Py and R, we're now trying to use snake case in both languages.

Reply:
@sebhrusen  and is it valid, that in py it is named rule_importance and in R there is get prefix : get_rule_importance ? or should it be the same?

Reply:
sorry, I read Pavel's question too quickly.
Ideally, for R properties, we shouldn't need any getter... I don't think we provide a getter for other model slots in R, do we? Do we do it here mainly to add the warning message?

Reply:
we created a hook for `stackedensemble` that allows to customize models for a specific algo:
see `kvstore.R`, line 266.
I think this could be used to add a slot to the RuleFit model only.

Reply:
otherwise, I don't see the benefit of having a new accessor in R, as it doesn't seem to add much value compared with the existing slot